tests/test_qaoa.py

```diff
@@ -1133,25 +1250,32 @@ def test_mixer_layer_output(self, mixer, gates):
         for i, j in zip(rec.operations, gates):
             prep = [i.name, i.parameters, i.wires]
             target = [j.name, j.parameters, j.wires]
+            assert prep == target
 
+    with qml.operation.disable_new_opmath_cm():
+        mixer_layer_test_cases_legacy = make_mixer_layer_test_cases()
+
+    @pytest.mark.usefixtures("use_legacy_opmath")
+    @pytest.mark.parametrize(("mixer", "gates"), mixer_layer_test_cases_legacy)
+    def test_mixer_layer_output_legacy_opmath(self, mixer, gates):
```

Comment:
I'm curious why you made a separate test for legacy op math rather than using the `use_legacy_and_new_opmath` fixture and converting to legacy H if old op math was active? Not a blocker and it doesn't really matter if you change it or not, but I noticed you used the `use_legacy_and_new_opmath` in a few tests above.

Reply:
Ah yes, for layer test cases I had to have two separate tests because legacy op math and new op math has inconsistent wire ordering of `PauliRot`, equivalent but different, which caused tests to fail.

Reply:
Ah right you mentioned this. Thanks!